It's a bit wired, still failed here, I dunno what's wrong

```
INFO: Potential tag based filters: {'bluetooth', 'cmsis_dsp', 'posix', 'net', 'mcumgr', 'test_framework', 'wifi', 'kernel'}
INFO: found boards: stm32h750b_dk,stm32h750b_dk/stm32h750xx/ext_flash_app,stm32h745i_disco/stm32h745xx/m7,stm32h745i_disco/stm32h745xx/m4
Traceback (most recent call last):
  File "/home/runner/work/zephyr/zephyr/zephyr/./scripts/ci/test_plan.py", line 549, in <module>
Changed files:
=========
boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7.dts
boards/st/stm32h750b_dk/stm32h750b_dk-common.dtsi
=========
    f.process()
  File "/home/runner/work/zephyr/zephyr/zephyr/./scripts/ci/test_plan.py", line 1[45](https://github.com/zephyrproject-rtos/zephyr/actions/runs/19567489845/job/56032927602?pr=99799#step:7:46), in process
    self.find_boards()
  File "/home/runner/work/zephyr/zephyr/zephyr/./scripts/ci/test_plan.py", line 292, in find_boards
    self.resolved_files.extend(list(filter(resolved_filter)))
                                    ^^^^^^^^^^^^^^^^^^^^^^^
TypeError: filter expected 2 arguments, got 1
Error: Process completed with exit code 1.
```

--- a/boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7.dts
+++ b/boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7.dts
@@ -37,7 +37,8 @@
 	sdram2: sdram@d0000000 {
 		compatible = "zephyr,memory-region", "mmio-sram";
 		device_type = "memory";
-		reg = <0xd0000000 DT_SIZE_M(16)>; /* 128Mbit */
+		/* 128Mbit, but only half is available by HW design */
+		reg = <0xd0000000 DT_SIZE_M(8)>;
 		zephyr,memory-region = "SDRAM2";
 		zephyr,memory-attr = <DT_MEM_ARM(ATTR_MPU_RAM)>;
 	};

--- a/boards/st/stm32h750b_dk/stm32h750b_dk-common.dtsi
+++ b/boards/st/stm32h750b_dk/stm32h750b_dk-common.dtsi
@@ -20,7 +20,8 @@
 	sdram2: sdram@d0000000 {
 		compatible = "zephyr,memory-region", "mmio-sram";
 		device_type = "memory";
-		reg = <0xd0000000 DT_SIZE_M(16)>; /* 128Mbit */
+		/* 128Mbit, but only half is available by HW design */
+		reg = <0xd0000000 DT_SIZE_M(8)>;
 		zephyr,memory-region = "SDRAM2";
 		zephyr,memory-attr = <DT_MEM_ARM(ATTR_MPU_RAM)>;
 	};

